### PR TITLE
Basic multi edition (Standard, BIM).

### PR DESCRIPTION
**Pull request by wielinde**
Independent:
- seed data
- design
- logo

Known, but won't fix now:
* [ ] Codeclimat warnings about long lines and duplicate code blocks. The long length is better for readability of table-like code. The duplicate code blocks will disappear once we fill it with real BIM seed data.

https://community.openproject.com/wp/29280

### CODE REVIEW

**Comment by oliverguenther** [comment]
It's going to be a big pain maintaining seeders for both configurations in one file. Please extract the seeders into `app/seeders/standard` and `app/seeders/bim` instead to use two separate sets of seeder classes (similary to BasicDataSeeder vs. DevelopmentDataSeeder) so you can request seeding them individually.

File: app/seeders/basic_data/type_seeder.rb [hunk 1]
```diff
@@ -65,19 +65,15 @@ def data
     end
 
     def type_names
-      %i[task milestone phase feature epic user_story bug]
+      if OpenProject::Configuration['edition'] == 'bim'
```
Comment (HDinger) [inline app/seeders/basic_data/type_seeder.rb]: Why is this not handled by the subclasses?

File: app/seeders/demo_data/project_seeder.rb [hunk 2]
```diff
@@ -201,4 +209,5 @@ def find_project(key)
 
     include Data
   end
+
```
Comment (HDinger) [inline app/seeders/demo_data/project_seeder.rb]: Can be removed and will at least remove one code climate issue

File: app/seeders/random_data/forum_seeder.rb [hunk 3]
```diff
@@ -33,9 +33,9 @@ def self.seed!(project)
       puts ''
       print ' ↳ Creating forum with posts'
 
-      forum = Forum.create! project: project,
-                            name: I18n.t('seeders.demo_data.board.name'),
-                            description: I18n.t('seeders.demo_data.board.description')
+      forum = Board.create! project: project,
```
Comment (HDinger) [inline app/seeders/random_data/forum_seeder.rb]: I guess this changes is incorrect, isn't it (perhaps a merging error)?

**Review by HDinger — APPROVED** [review]

**Comment by HDinger** [comment]
@wielinde Please have a look at the things I commented and whether the failing tests have something to with your changes. Otherwise good to merge 👍

File: app/seeders/demo_data/project_seeder.rb [hunk 4]
```diff
@@ -156,11 +160,15 @@ def seed_versions(project, key)
...
         project: project,
-        name: translate_with_base_url('seeders.demo_data.board.name'),
-        description: translate_with_base_url('seeders.demo_data.board.description')
+        name: translate_with_base_url("seeders.#{OpenProject::Configuration['edition']}.demo_data.board.name"),
+        description: translate_with_base_url("seeders.#{OpenProject::Configuration['edition']}.demo_data.board.description")
       )
     end
 
+    def project_key(key)
+      "seeders.#{OpenProject::Configuration['edition']}.demo_data.projects.#{key}"
```
Comment (HDinger) [inline app/seeders/demo_data/project_seeder.rb]: This line is written very often in this changes. Please consider making it a public method in the super class or at least use the methods you defined.

**Review by HDinger — COMMENTED** [review]
Hi @wielinde 

Is there a reason you did not move all the bim stuff to the new module, e.g. the seeder.yml?

File: lib/open_project/design.rb [hunk 5]
```diff
@@ -200,7 +200,22 @@ class Design
       'table-timeline--row-height'                           => '40px',
       'status-selector-bg-color'                             => '#F99601',
       'status-selector-bg-hover-color'                       => '#E08600'
-    }.freeze
+    }.merge(
+      if OpenProject::Configuration['edition'] == 'bim'
```
Comment (HDinger) [inline lib/open_project/design.rb]: This might be better in the module, too

**Comment by wielinde** [comment]
@HDinger 
- I hopefully removed every BIM specific logic, assets and checks from the core. 
- Further, I DRYed up the I18n key composition.
- Extended our Travis configuration to check the two editions in combination with postgres.

**Review by HDinger — APPROVED** [review]

**Pull request merged**